mautic/mautic
# Single click tag deletion

5.x

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [✖️]
| New feature/enhancement? (use the a.x branch)      | [✔️]
| Deprecations?                          | [✖️]
| BC breaks? (use the c.x branch)        | [✖️]
| Automated tests included?              |  [✖️] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR provides way to delete tags directly in contact view by clicking on cross 

https://github.com/mautic/mautic/assets/96595532/cf9ba70f-9132-4217-8121-fd25e98a99e5



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new tag
3. Create a new contact and add to him recently created tag or tags
4. You see cross on contact's tag which you can click and it will be removed
